test/Microsoft.AspNet.Razor.Runtime.Test/TagHelperContentWrapperTextWriterTest.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Text;
+using Xunit;
+
+namespace Microsoft.AspNet.Razor.Runtime
+{
+    public class TagHelperContentWrapperTextWriterTest
```

Comment:
Depending on your response to what I said above this class may not be necessary.